### ISSUE TITLE
value too large

### ISSUE THREAD

**Issue by spirillen** [funilrys/PyFunceble#270]
## Description

<details><summary>stdout output</summary>

```python
www.courtingyou.com                                                                                  ACTIVE      DNSLOOKUP  Unknown           302        AVAILABILITY 
Process pyfunceble_tester_worker_3:
Traceback (most recent call last):
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/rdata.py", line 649, in from_wire_parser
    return cls.from_wire_parser(rdclass, rdtype, parser, origin)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/rdtypes/ANY/SOA.py", line 78, in from_wire_parser
    return cls(rdclass, rdtype, mname, rname, *parser.get_struct('!IIIII'))
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/_immutable_ctx.py", line 41, in nf
    f(*args, **kwargs)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/rdtypes/ANY/SOA.py", line 44, in __init__
    self.expire = self._as_ttl(expire)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/rdata.py", line 446, in _as_ttl
    return cls._as_int(value, 0, dns.ttl.MAX_TTL)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/rdata.py", line 411, in _as_int
    raise ValueError('value too large')
ValueError: value too large

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/cli/processes/workers/base.py", line 388, in run
    raise exception
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/cli/processes/workers/base.py", line 366, in run
    result = self.target(consumed)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/cli/processes/workers/tester.py", line 276, in target
    .query_status()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/base.py", line 157, in wrapper
    return func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/base.py", line 182, in wrapper
    result = func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/availability/domain_and_ip.py", line 109, in query_status
    result = query_object.query_status()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/base.py", line 157, in wrapper
    return func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/base.py", line 182, in wrapper
    result = func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/availability/domain.py", line 169, in query_status
    self.try_to_query_status_from_http_status_code()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/availability/base.py", line 870, in try_to_query_status_from_http_status_code
    lookup_result = self.http_status_code_query_tool.get_status_code()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/http_status_code.py", line 125, in wrapper
    return func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/http_status_code.py", line 340, in get_status_code
    req = PyFunceble.factory.Requester.get(
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/requests/requester.py", line 157, in wrapper
    req = getattr(self.session, verb.lower())(*args, **kwargs)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/requests/sessions.py", line 555, in get
    return self.request('GET', url, **kwargs)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/requests/adapter/http.py", line 92, in send
    hostname_ip = self.resolve(parsed_url.hostname)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/requests/adapter/base.py", line 181, in resolve
    return self.resolve_without_cache(hostname)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/requests/adapter/base.py", line 153, in resolve_without_cache
    last_cname = get_last_cname(hostname)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/requests/adapter/base.py", line 132, in get_last_cname
    .query()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/dns/query_tool.py", line 1048, in query
    return getattr(self, self.preferred_protocol.lower())()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/dns/query_tool.py", line 230, in wrapper
    return func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/dns/query_tool.py", line 260, in wrapper
    return func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/dns/query_tool.py", line 242, in wrapper
    result = func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/dns/query_tool.py", line 848, in udp
    response = dns.query.udp(
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/query.py", line 505, in udp
    (r, received_time) = receive_udp(s, destination, expiration,
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/query.py", line 445, in receive_udp
    r = dns.message.from_wire(wire, keyring=keyring, request_mac=request_mac,
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/message.py", line 1074, in from_wire
    m = reader.read()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/message.py", line 999, in read
    self._get_section(MessageSection.AUTHORITY, aucount)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/message.py", line 938, in _get_section
    rd = dns.rdata.from_wire_parser(rdclass, rdtype,
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/rdata.py", line 648, in from_wire_parser
    with dns.exception.ExceptionWrapper(dns.exception.FormError):
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/exception.py", line 141, in __exit__
    raise self.exception_class(str(exc_val)) from exc_val
dns.exception.FormError: value too large
www.couturehollywood.com                                                                             ACTIVE      DNSLOOKUP  Unknown           301        AVAILABILITY 
Process pyfunceble_tester_worker_5:
Traceback (most recent call last):
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/rdata.py", line 649, in from_wire_parser
    return cls.from_wire_parser(rdclass, rdtype, parser, origin)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/rdtypes/ANY/SOA.py", line 78, in from_wire_parser
    return cls(rdclass, rdtype, mname, rname, *parser.get_struct('!IIIII'))
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/_immutable_ctx.py", line 41, in nf
    f(*args, **kwargs)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/rdtypes/ANY/SOA.py", line 44, in __init__
    self.expire = self._as_ttl(expire)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/rdata.py", line 446, in _as_ttl
    return cls._as_int(value, 0, dns.ttl.MAX_TTL)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/rdata.py", line 411, in _as_int
    raise ValueError('value too large')
ValueError: value too large

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/cli/processes/workers/base.py", line 388, in run
    raise exception
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/cli/processes/workers/base.py", line 366, in run
    result = self.target(consumed)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/cli/processes/workers/tester.py", line 276, in target
    .query_status()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/base.py", line 157, in wrapper
    return func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/base.py", line 182, in wrapper
    result = func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/availability/domain_and_ip.py", line 109, in query_status
    result = query_object.query_status()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/base.py", line 157, in wrapper
    return func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/base.py", line 182, in wrapper
    result = func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/availability/domain.py", line 153, in query_status
    self.try_to_query_status_from_dns()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/availability/base.py", line 795, in try_to_query_status_from_dns
    lookup_result = self.query_dns_record()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/base.py", line 157, in wrapper
    return func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/checker/availability/base.py", line 683, in query_dns_record
    ).query()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/dns/query_tool.py", line 1048, in query
    return getattr(self, self.preferred_protocol.lower())()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/dns/query_tool.py", line 230, in wrapper
    return func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/dns/query_tool.py", line 260, in wrapper
    return func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/dns/query_tool.py", line 242, in wrapper
    result = func(self, *args, **kwargs)  # pylint: disable=not-callable
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/query/dns/query_tool.py", line 848, in udp
    response = dns.query.udp(
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/query.py", line 505, in udp
    (r, received_time) = receive_udp(s, destination, expiration,
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/query.py", line 445, in receive_udp
    r = dns.message.from_wire(wire, keyring=keyring, request_mac=request_mac,
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/message.py", line 1074, in from_wire
    m = reader.read()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/message.py", line 999, in read
    self._get_section(MessageSection.AUTHORITY, aucount)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/message.py", line 938, in _get_section
    rd = dns.rdata.from_wire_parser(rdclass, rdtype,
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/rdata.py", line 648, in from_wire_parser
    with dns.exception.ExceptionWrapper(dns.exception.FormError):
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/dns/exception.py", line 141, in __exit__
    raise self.exception_class(str(exc_val)) from exc_val
dns.exception.FormError: value too large
covabrelli.com                                                                                       ACTIVE      WHOIS      21-mar-2022       301        AVAILABILITY 
www.covabrelli.com                                                                                   ACTIVE      DNSLOOKUP  Unknown           301        AVAILABILITY 
Fatal Error: <self.subject> should be <class 'str'>, <class 'NoneType'> given.
Traceback (most recent call last):
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/cli/system/launcher.py", line 918, in start
    self.stop_and_wait_for_all_manager()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/cli/system/launcher.py", line 855, in stop_and_wait_for_all_manager
    self.tester_process_manager.wait()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/cli/processes/base.py", line 320, in wait
    raise worker_error
TypeError: <self.subject> should be <class 'str'>, <class 'NoneType'> given.

covalentcrm.com                                                                                      ACTIVE      WHOIS      22-oct-2022       301        AVAILABILITY 
Exception during reset or similar
Traceback (most recent call last):
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/pymysql/connections.py", line 756, in _write_bytes
    self._sock.sendall(data)
BrokenPipeError: [Errno 32] Broken pipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 739, in _finalize_fairy
    fairy._reset(pool)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 988, in _reset
    pool._dialect.do_rollback(self)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 667, in do_rollback
    dbapi_connection.rollback()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/pymysql/connections.py", line 479, in rollback
    self._execute_command(COMMAND.COM_QUERY, "ROLLBACK")
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/pymysql/connections.py", line 814, in _execute_command
    self._write_bytes(packet)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/pymysql/connections.py", line 759, in _write_bytes
    raise err.OperationalError(
pymysql.err.OperationalError: (2006, "MySQL server has gone away (BrokenPipeError(32, 'Broken pipe'))")
Exception ignored in: <function SystemLauncher.__del__ at 0x7f1780d835b0>
Traceback (most recent call last):
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/PyFunceble/cli/system/launcher.py", line 254, in __del__
    self.db_session.close()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 1786, in close
    self._close_impl(invalidate=False)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 1828, in _close_impl
    transaction.close(invalidate)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 923, in close
    transaction.close()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 2315, in close
    self._do_close()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 2531, in _do_close
    self._close_impl()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 2517, in _close_impl
    self._connection_rollback_impl()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 2509, in _connection_rollback_impl
    self.connection._rollback_impl()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 965, in _rollback_impl
    self._handle_dbapi_exception(e, None, None, None, None)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 2023, in _handle_dbapi_exception
    util.raise_(
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
    raise exception
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 963, in _rollback_impl
    self.engine.dialect.do_rollback(self.connection)
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 667, in do_rollback
    dbapi_connection.rollback()
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/pymysql/connections.py", line 479, in rollback
    self._execute_command(COMMAND.COM_QUERY, "ROLLBACK")
  File "$HOME/miniconda/envs/pyfunceble-dev/lib/python3.10/site-packages/pymysql/connections.py", line 793, in _execute_command
    raise err.InterfaceError(0, "")
sqlalchemy.exc.InterfaceError: (pymysql.err.InterfaceError) (0, '')
(Background on this error at: https://sqlalche.me/e/14/rvf5)
```

<details><summary>.PyFunceble.overwrite.yaml</summary>

## Configuration
</details>
cli_testing:
  cidr_expand: true
  db_type: mariadb
  display_mode:
    all: true
    colour: true
    dots: false
    execution_time: true
    less: false
    percentage: true
    quiet: false
    simple: false
    status: all
  file_generation:
    analytic: true
    hosts: false
    merge_output_dirs: false
    no_file: false
    plain: true
    unified_results: false
  local_network: true
  sorting_mode:
    hierarchical: true
debug:
  active: false
  level: critical
lookup:
  dns: true
  http_status_code: true
  netinfo: true
  reputation: false
  special: true
  timeout: 5
  whois: true
user_agent:
  browser: firefox
  custom: null
  platform: linux
dns:
  follow_server_order: false
  trust_server: false
  protocol: UDP
  server: # https://mypdns.org/-/snippets/41
    - 130.226.161.34
    - 89.233.43.71
    - 185.121.177.177
    - 163.172.168.171
    - 192.168.1.100:5302
    - 192.168.1.6
    - 9.9.9.10
    - 149.112.112.10
collection:
   push: true
   url_base: https://collection.dead-hosts.funilrys.com
</details>

Posting this log make me wish this Feature request wasn't closed unsolved https://github.com/funilrys/PyFunceble/issues/141

## Reproduction

1. One huge RPZ zone ie: `drill axfr @ns1.mypdns.org -p 5303 rpz.mypdns.cloud > rpz.mypdns.cloud`
2. git clone https://mypdns.org/pyfunceble/pyfunceble-templates/pyfunceble-miniconda.git
3. Execute pyfunceble.sh rpz.mypdns.cloud
4. prepend `--rpz` to the test arguments
5. Wait ~48 hours...

## Screenshots
Not neede

## Versions

**OS:** Ubuntu 20.04 5.4.0-90-generic

**Python Version:** 3.10

**PyFunceble Version:** [pyfunceble-dev>=4.0.2b1](https://mypdns.org/pyfunceble/pyfunceble-templates/pyfunceble-miniconda/-/blob/11b4d09f34801b9b23d4fa7e322dc0f8f4d9dbab/.environment.pyfunceble-dev.yaml)

**Comment by spirillen** [comment]
This epoch error reappeared after running the above test for about 8-10 days....

See the Y2k38 bug https://github.com/funilrys/PyFunceble/issues/261

```
sqlalchemy.exc.DataError: (pymysql.err.DataError) (1264, "Out of range value for column 'epoch' at row 1")
[SQL: INSERT INTO pyfunceble_whois_record (created_at, modified_at, subject, idna_subject, expiration_date, epoch) VALUES (%(created_at)s, %(modified_at)s, %(subject)s, %(idna_subject)s, %(expiration_date)s, %(epoch)s)]
[parameters: {'created_at': datetime.datetime(2021, 12, 2, 6, 47, 43, 836904), 'modified_at': datetime.datetime(2021, 12, 2, 6, 47, 43, 836919), 'subject': 'edu.ge', 'idna_subject': 'edu.ge', 'expiration_date': '02-jan-2099', 'epoch': '4070991600.0'}]
(Background on this error at: https://sqlalche.me/e/14/9h9h)
```

**Comment by funilrys** [comment]
Should be fixed with #261.
